app/dashboard/models.py

```diff
@@ -1311,6 +1313,19 @@ def update_slack_integration(self, token, channel, repos):
         self.slack_channel = channel
         self.save()
 
+    def update_discord_integration(self, webhook_url, repos):
+        """Update the profile's Discord integration settings.
+
+        Args:
+            webhook_url (str): The profile's Discord webhook url.
+            repos (list of str): The profile's github repositories to track.
+
+        """
+        repos = repos.split()
```

Comment:
dont you have to split it on a char, like `','`?

Reply:
Yes, I missed this - fixed the same for update_slack_integration but you may have to migrate existing slack_repos data as I suspect they would have been saved as ["slack_repo_1,slack_repo_2"] instead of ["slack_repo_1", "slack_repo_2"].